src/style.css

```diff
@@ -1,3 +1,103 @@
+:root {
+  --color-hover: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+}
+
+/* ul {
```

Comment:
The code from line 16 to 25 is commented out. If this code is not needed, consider removing it to keep the code base clean.